app/js/templates/adminTable.hbs

```diff
@@ -16,10 +18,17 @@
      <td class="email">{{email}}</td>
      <td class="percentage">{{percentage}}</td>
      <td class="date"></td>
+     {{#if autoCompleted}}
+     <td class="autocomplete-icon"><i class="icon-autocomplete bi bi-exclamation-diamond"></i>
```

Comment:
can you add a title attribute to the `<i>` with a "Autocompleted" value, so that it shows a little description of what this icon means